src/mc.rs

```diff
@@ -101,6 +102,21 @@ pub fn query_public_addr(
     }
 }
 
+pub fn enable_igd_for_rendezvous() {
+    let mut mc = unwrap!(MC.lock());
```

Comment:
one thing I don't really like is that this is a global option affecting every call to `p2p`.
Although, currently this seems ok, but in the long term I'd like to be able to configure this for each connection.
Or event better have separate calls, e.g. `TcpStreamExt::rendezvous_connect()` and `TcpStreamExt::igd_connect()`
Anyway, when I say it out loud, I start to doubt if IGD is even desired here...